### PR TITLE
feat: update translation key create/update with custom metadata. Add CM schema in translation key response

### PR DESCRIPTION
**Pull request by anthonyshchang**
### Description
update translation key create/update with custom metadata. Also. add CM in translation key schema response

response CM in translation key
Screenshot 2023-12-12 at 14 20 33

Update/Create CM on translation key

https://github.com/phrase/openapi/assets/108525179/cecd517f-6d0e-42fc-a172-812c80612d69

### Ticket
https://phrase.atlassian.net/browse/TSS-2504

### CODE REVIEW

**Review by malvarezphrase — APPROVED** [review]

File: paths/keys/update.yaml [hunk 1]
```diff
@@ -120,4 +121,10 @@ requestBody:
             description: NSStringLocalizedFormatKey attribute. Used in .stringsdict format.
             type: string
             example:
+          custom_metadata:
+            description: Custom metadata property name and value pairs to be associated with key.
```
Comment (jablan) [inline paths/keys/update.yaml:125]: I believe it should be mentioned what happens with existing CM values on this key, because I have no idea. :)

**Review by jablan — CHANGES_REQUESTED** [review]
docs for update should be clarified IMO

File: paths/keys/update.yaml (hunk 1 above)
Comment (ildarsafin) [reply paths/keys/update.yaml]: @jablan is it more descriptive now? :)

**Comment by forelabs** [comment]
Will have a look in a second

**Review by forelabs — APPROVED** [review]

**Review by jablan — APPROVED** [review]

**Pull request merged**